packages/components/src/components/list-box/_list-box.scss

```diff
@@ -498,13 +502,15 @@ $list-box-menu-width: rem(300px);
     }
   }
 
-  .#{$prefix}--list-box--disabled
-    .#{$prefix}--list-box__selection--multi
+  .#{$prefix}--list-box--disabled .#{$prefix}--list-box__selection--multi {
+    @include tag-theme($disabled-02, $disabled-01);
```

Comment:
Seems like `tag-theme` is not imported in `_list-box.scss` and is causing some build errors. 

We'll need to add a `@import '../tag/mixins';` to the top of `_list-box.scss`